Debugger/src/DebuggerClient.php

```diff
@@ -39,7 +40,7 @@ class DebuggerClient
     use ClientTrait;
 
     const VERSION = '0.9.0';
-    const DEFAULT_AGENT_VERSION = 'google.com/gcp-php/v0.1';
+    const DEFAULT_AGENT_VERSION = 'google.com/gcp-php/v0.9';
```

Comment:
I liked what you were going for by appending the VERSION constant, as this will definitely continue to fall out of sync. A workaround we could use would be to use a static function:

```php
public static function getDefaultAgentVersion()
{
    return 'google.com/gcp-php/' . self::VERSION;
}
```

WDYT?

Reply:
Makes sense, we'll go with that. It's a tiny bit of overhead that we can remove if we ever stop supporting 5.5.

Reply:
Done.